### DISCUSSION TITLE
Parameter Identification

### DISCUSSION THREAD

**Discussion by joao-reis3** [Open-Systems-Pharmacology/Forum#873] [Questions & Problems]
Hello everyone,

My team's research focuses on generic development, and we are having trouble predicting the _in vivo_ performance of one of our formulations.
I'm using PK-Sim to predict the plasma-concentration of a BCS IV drug with a different amount of a certain excipient (Poloxamer) which really affects its solubility and absorption. 
We have data (_in vivo_ and dissolution profiles) from 3 other formulations (different amounts of poloxamer) but we don't have values for the formulations' permeability and clearance. As such, we are using the parameter identification tool to predict all missing values and trying to find tendencies (e.g., linear permeability with increasing excipient concentration) to predict our formulation's _in vivo_ performance.

The problem is that we are having trouble using the parameter identification tool - our simulations do not fit the observable data by 100%, which lead us to poor trustable conclusions. **Which is the best way to use the parameter identification tool?**

We would also be grateful if anyone could point us in another direction if you think this is not the best way to answer our research question.

**Comment by fredsevmartins** [comment]
Hello @joao-reis3 
I do not think it is a problem with the PI tool. 
-Try to fit only key parameters or use sensitivity analysis to select the most important one. 
-Make sure that you have the accurate dissolution method to IVIVE-PBPK
-Does the Polaxamer play with intestinal permeability or only solubility !?
-Did you incorporated the transporters and enzyme metabolism ?

Please contact me by email if you need more support :Frederico.martins@esqlabs.com

**Comment by Ricardo-DO** [comment]
Hi Joao, 

If you have IV in vivo data of the compound, you can get the total plasma clearance from and NCA and add it to your simulation (assuming linear PK).
Then you can optimise the permeability, you set your initial values around 1 x 10^-4 cm/min.
Hope that works.

Regards,
Ricardo

**Comment by joao-reis3** [comment]
Hello @Ricardo-DO 

Thank your for your answer. 
I have no IV in vivo data, when I said BCS IV, I meant BCS class 4 (low solubility and low permeability).

I only have in vivo data from 3 oral formulations. I tried to calculate clearance but since my compound is low permeable, I achieved a really low value (which doesn't mimic what happens in vivo).

Regards,
João
